Cargo.toml

```diff
@@ -15,8 +15,10 @@ ctr = { version = "0.9.2", optional = true }
 cmac = { version = "0.7.1", optional = true }
 base64 = "0.20.0"
 byteorder = "1.4.3"
+fiat-crypto = "0.1.17"
 fixed = { version = "1.21", optional = true }
 getrandom = { version = "0.2.8", features = ["std"] }
+num-bigint = "0.4.3"
```

Comment:
Is it feasible to move this to `dev-dependencies`? See comment on `Display` for `Field255`.